dispatchers.go

```diff
@@ -606,6 +606,7 @@ func healthcheck(b *Builder, args []string, attributes map[string]bool, flagArgs
 
 		flags := flag.NewFlagSet("", flag.ContinueOnError)
 		flags.String("start-period", "", "")
+		flags.String("start-interval", "", "")
```

Comment:
The reference link specifies defaults for these settings.  Are we returning those defaults when there isn't a corresponding value supplied in the instruction?

Reply:
Let me add a default value here.

Reply:
Oof, I guess the real question should have been whether those defaults get baked into the image, or if they're deferred until run-time.  I _think_ a conformance test that uses a HEALTHCHECK instruction that leaves the various settings at their defaults would tell us whether or not the defaults should be provided in the Config in the committed image.

Reply:
I have a hunch that `docker` should be doing this on runtime end and not baking into images, otherwise our previous conformance tests would have failed since we never had defaults. Let me try running the conformance test to verify this.

Reply:
I'm not sure we have a conformance test for that (I see a unit test that uses Dockerfile.healthcheck and Dockerfile.badhealthcheck, but that mainly exercises the parsing logic), and if you don't see one, either, it'd probably be a good idea to add one.

Reply:
Okay, good to know.  Thanks for verifying!